hack/build-rpms.sh

```diff
@@ -0,0 +1,131 @@
+#!/bin/bash
```

Comment:
Total noob question, but why are we doing this instead of using tito to do local rpm builds?

Wouldn't using tito for local rpm builds ensure that pushed code will build in the ci?

Reply:
Couldn't tell you. This code is from [the standard toolchain](https://github.com/openshift/release/tree/master/tools/hack/golang); probably want to ask the folks who maintain it.

From my perspective as a developer of the app, all I care about is that I can:

* Build binaries natively (with `go`)
* Build an image with `docker` (the fact that RPM is used in that pipeline is an irrelevant detail to me)

AFAIK the fact that I can run `OS_BUILD_ENV_PRESERVE=_output/local/bin hack/env make build-images` is proof enough that what I have is likely to build in CI... correct me if I'm wrong

Reply:
Ah, ok, I wasn't aware of the "standard toolchain". It makes sense that you'd want it here then.